nemo/export/quantize/quantizer.py

```diff
@@ -167,23 +100,17 @@ def dummy():
         set_data_parallel_group(mpu.get_data_parallel_group())
         set_tensor_parallel_group(mpu.get_tensor_model_parallel_group())
 
-    @staticmethod
-    def _load_and_modify_config(
-        model_file: str,
-        tensor_model_parallel_size: Optional[int] = None,
-        pipeline_model_parallel_size: Optional[int] = None,
-    ):
-        model_cfg = load_config(model_file)
+        if inference_config is not None:
+            model.set_inference_config(OmegaConf.to_container(inference_config))
 
+    @staticmethod
+    def modify_model_config(model_cfg: DictConfig) -> DictConfig:
+        """Modify model config for quantization."""
         with open_dict(model_cfg):
             model_cfg.activations_checkpoint_method = None
             model_cfg.activations_checkpoint_granularity = None
```

Comment:
Can we remove this disable of activation checkpointing as well? For the SFT model there is no accuracy regression even if we enable activation checkpointing plus that reduces GPU memory requirement

Reply:
I'm not sure regarding this, see also related examples in https://github.com/NVIDIA/NeMo/pull/9276#discussion_r1624463595. Maybe you could ask internally?

Reply:
So in the end it turns out that it's OK to remove setting these to `None`?:
```
model_cfg.activations_checkpoint_method = None
model_cfg.activations_checkpoint_granularity = None
```
I hope it will not blow up in some large tests

Reply:
Users always have the option to disable it in the yaml if they want to, but we should not restrict them to not be able to use it as it allows reducing GPU memory requirement

Reply:
I will set them as None in the yaml file so there is no difference than previous version